lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -1133,13 +1109,13 @@ PrimUpdaterManager& PrimUpdaterManager::getInstance()
 MObject PrimUpdaterManager::findOrCreatePullRoot()
 {
     MObject pullRoot = findPullRoot();
-    if (!pullRoot.isNull())
+    if (!pullRoot.isNull()) {
+        _hasPulledPrims = true;
```

Comment:
Why is this line needed?  I thought that the commands below would set it correct.  Shouldn't this rather be
TF_AXIOM(_hasPulledPrims);
?

Reply:
Ah yes, I'll remove this.